trainset_preprocess_pipeline_print.py

```diff
@@ -110,31 +92,16 @@ def pipeline_mp_inp_dir(self, inp_root, n_p):
                 ("%s/%s" % (inp_root, name), idx)
                 for idx, name in enumerate(sorted(list(os.listdir(inp_root))))
             ]
-            if noparallel:
-                for i in range(n_p):
-                    self.pipeline_mp(infos[i::n_p])
-            else:
-                ps = []
-                for i in range(n_p):
-                    p = multiprocessing.Process(
-                        target=self.pipeline_mp, args=(infos[i::n_p],)
-                    )
-                    ps.append(p)
-                    p.start()
-                for i in range(n_p):
-                    ps[i].join()
+            for i in range(n_p):
+                self.pipeline_mp(infos[i::n_p])
         except:
             raise Exception("Fail. %s" % traceback.format_exc())
 
 
 
 def preprocess_trainset(inp_root, sr, n_p, exp_dir):
     pp = PreProcess(sr, exp_dir)
-    println("start preprocess")
-    println(sys.argv)
+    log.info("start preprocess")
+    log.info(sys.argv)
```

Comment:
remove this line.
you're not calling it from command line any more

Reply:
Done.